Controller/Callback/Offsite.php

```diff
@@ -76,8 +76,10 @@ public function execute()
 
             return $this->redirect(self::PATH_CART);
         }
-
-        if ($payment->getMethod() !== 'omise_offsite_internetbanking') {
+        
+        $paymentMethod = $payment->getMethod();
+        
+        if ($paymentMethod !== 'omise_offsite_internetbanking' && $paymentMethod !== 'omise_offsite_alipay' ) {
```

Comment:
@jacekstan Just idea, but maybe this can make code a bit cleaner
```php
use Omise\Payment\Model\Config\Offsite\Alipay;
use Omise\Payment\Model\Config\Offsite\Internetbanking;

...

if (! in_array($payment->getMethod(), array(Alipay::CODE, Internetbanking::CODE)) {
    // ...
}
```